### DISCUSSION TITLE
TriangleAdapt feasibility

### DISCUSSION THREAD

**Discussion by JesterWizard** [MokhaLeee/fe8u-cskillsys-kernel#220] [General]
I wanted to make a start on the TriangleAdapt skills next, but I know you had some reservations regarding multiplication in that part of the code. 

Am I okay to make a start on them, or should I wait until it's better optimized?

**Comment by MokhaLeee** [comment]
To be honest, I don't support placing more burdens on WTA. Because that has already been the worst part of the code in battle sys on considering of performance and will be rewritten latter. I think you can add skill info first and comment out the skill index. later on I may directly realize it

**Comment by JesterWizard** [comment]
This is done now, so closing.

**Comment by MokhaLeee** [comment]
Oh, I have to withdraw my previous views. Previous I thought that since the WTA calc as a loop, thus a O(n) level MUL operation is unacceptable. But now I"m aware that each loop hold a "break", then it doesn't matter if the internal operation is more complicated.
